client/bot-community.py

```diff
@@ -13,30 +13,34 @@
    ./bot-community.py --port 9945 init
    ./bot-community.py --port 9945 benchmark
    
-on testnet Gesell, run this script once per ceremony phase (after calling `init` first)
-   ./bot-community.py --port 9945 run
+on testnet Gesell, execute the current ceremony phase (it does not advance the phase).
+   ./bot-community.py --port 9945 execute-current-phase
 
 """
 import os
 
 import click
 import geojson
+import ast
 
 from random_words import RandomWords
 from math import floor
 
 from py_client.helpers import purge_prompt, read_cid, write_cid, zip_folder, set_local_or_remote_chain
-from py_client.client import Client, ExtrinsicFeePaymentImpossible, ExtrinsicWrongPhase, UnknownError, ParticipantAlreadyLinked
+from py_client.client import Client, ExtrinsicFeePaymentImpossible, ExtrinsicWrongPhase, UnknownError, \
+    ParticipantAlreadyLinked
 from py_client.ipfs import Ipfs, ICONS_PATH
 from py_client.communities import populate_locations, generate_community_spec, meta_json
 
 KEYSTORE_PATH = './my_keystore'
 NUMBER_OF_LOCATIONS = 100
 MAX_POPULATION = 12 * NUMBER_OF_LOCATIONS
+NUMBER_OF_ENDORSEMENTS_PER_REGISTRATION = 101
```

Comment:
I reckon we will fine tune this afterwards. The script does not support meetups anyhow until we integrated handling of randomized meetups.